pinot-common/src/main/java/org/apache/pinot/pql/parsers/pql2/ast/SelectAstNode.java

```diff
@@ -106,7 +106,7 @@ public String toString() {
   public void updateBrokerRequest(BrokerRequest brokerRequest) {
     // Set the query source
     QuerySource querySource = new QuerySource();
-    querySource.setTableName(_resourceName);
+    querySource.setTableName(_tableName);
```

Comment:
I think we don't differentiate table name and resource name right now, so maybe a better fix should be in `TableNameAstNode` to remove the `resourceName` and only keep the `tableName`?

Reply:
Also curious about how it causes the failure for PQL. I don't see any table name with dot embedded

Reply:
So before this PR, if the query is like `SELECT myDB.myTable FROM ...`, the table name would be parsed as `myDB` instead of `myTable`. In this case, 

- For getting table name from table cache in broker, since there is no table called `myDB`, the table name won't be adjusted in `updateTableName` method of `BaseBrokerRequestHandler` class.
- In server, since there is no table called `myDB`, an empty response would be returned.

Table name with dot embedded is actually supported right now. You can refer to the calcite parser logic and the `testCaseInsensitivity` of `OfflineClusterIntegrationTest`.

Reply:
I see. You might also want to use `_tableName` on line 172 for updating the pinot query

Reply:
Thanks. Addressed.